client/src/modules/CrossCheck/EditableCellForCrossCheck.tsx

```diff
@@ -1,42 +1,44 @@
-import { Form, InputNumber, Input, Select } from 'antd';
+import { Form, InputNumber, Input } from 'antd';
 import { CriteriaDto } from 'api';
 import React from 'react';
+import { TaskType } from './components/CrossCheckCriteriaForm';
 
 const { TextArea } = Input;
 
 interface EditableCellProps extends React.HTMLAttributes<HTMLElement> {
   editing: boolean;
   dataIndex: string;
-  title: any;
   inputType: 'max' | 'text' | 'description';
   record: CriteriaDto;
   index: number;
   children: React.ReactNode;
+  type: TaskType;
+  points: undefined | number;
 }
 
 export const EditableCellForCrossCheck: React.FC<EditableCellProps> = ({
   editing,
   dataIndex,
   inputType,
   children,
+  type,
+  points,
   ...restProps
 }) => {
   const inputNode =
-    inputType === 'max' ? (
-      <InputNumber style={{ width: 65 }} />
-    ) : inputType === 'text' ? (
-      <Select style={{ width: 75 }}>
-        <Select.Option value="Title">Title</Select.Option>
-        <Select.Option value="Subtask">Subtask</Select.Option>
-        <Select.Option value="Penalty">Penalty</Select.Option>
-      </Select>
-    ) : (
+    inputType === 'description' ? (
       <TextArea rows={3} />
-    );
+    ) : type !== TaskType.Title ? (
+      <InputNumber style={{ width: 65 }} />
+    ) : null;
 
   return (
-    <td {...restProps}>
-      {editing ? (
+    <td
+      {...restProps}
+      title={points === 0 ? 'Check points for this line' : ''}
```

Comment:
minor, but i would extract `points === 0` into a variable

Reply:
Thanks for the review!
Fixed it